analytical/templatetags/piwik.py

```diff
@@ -21,20 +21,22 @@
 SITEID_RE = re.compile(r'^\d+$')
 
 TRACKING_CODE = """
+<!-- Piwik -->
```

Comment:
I see this is in the [official Piwik sample](https://developer.piwik.org/guides/tracking-javascript-guide). For me, comments in HTML are a bad habit (from a time when PHP was the awesome child). If we need comments at all we can use Python comments or a comment tag in the template.

Let's not include HTML comments in our code. (Unless we really need help to analyze our HTML output.)